test/unit/karma.conf.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (config) {
         },
         files: [
             'kbase-extension/static/narrative_paths.js',
-            {pattern: 'test/unit/spec/**/*.js', included: false},
+            {pattern: 'test/unit/spec/nbextensions/**/*.js', included: false},
```

Comment:
Is this suppose to be checked in?

Reply:
I coulda sworn I fixed that... Thanks for catching!